common/source/docs/common-lua-scripts.rst

```diff
@@ -254,13 +286,58 @@ The GPS library provides access to information about the GPS's on the vehicle.
 
 - :code:`first_unconfigured_gps()` - Returns nil or the instance number of the first GPS that has not been fully configured. If all GPS's have been configured this returns 255 if all the GPS's have been configured.
 
-Notify
-~~~~~~
+GCS (gcs:)
+~~~~~~~~~~
+
+- :code:`send_text(severity, "text")` - Will send the text variable or constant string with message severity . Severity level is :
+
++---------------+-----------+
+|Severity Level | Type      |
++---------------+-----------+
+|0              | Emergency |
++---------------+-----------+
+|1              | Alert     |
++---------------+-----------+
+|2              | Critical  |
++---------------+-----------+
+|3              | Error     |
++---------------+-----------+
+|4              | Warning   |
++---------------+-----------+
+|5              | Notice    |
++---------------+-----------+
+|6              | Info      |
++---------------+-----------+
+
+
+- :code:`set_message_interval(serial channel, message type, rate)` - Sets the message types update rate on SERIAL (serial channel). For example, gcs:set_message_interval(0, uint32_t(30), math.floor(1000000 / 2.0)) sets SERIAL0 rate for ATTITUDE message stream (30) to 2.0 Hz .
+
+Serial LED (serialLED:)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This library allows the control of WS8212B RGB LED strings via an output reserved for scripting and  selected by SERVOx_FUNCTION = 94 thru 109 (Script Out 1 thru 16)
+
+- :code:`set_num_LEDS(output number, number of LEDs)` - Sets servo output number and number of LEDs in the string on that output. "output number" is servo output number 1-16 that the string is attached with a string having "number of LEDs".
+
+- :code:`set_RGB(output number, LED number, r,g,b)` - Set the data for "LED number" (1-32) on "output number" (1-16) output the the "r,g,b" values (0-255)
+
+- :code:`send()` - Sends the data to the LED strings
+
+
+Notify (notify:)
+~~~~~~~~~~~~~~~~
 
 - :code:`play_tune(tune)` - Plays a MML tune through the buzzer on the vehicle. The tune is provided as a string.
 
-Terrain
-~~~~~~~
+
+Vehicle (vehicle)
+~~~~~~~~~~~~~~~~~
+
+- :code:`set_mode(mode number)` - Attempts to change vehicle mode to "mode number". Returns true if successful.
```

Comment:
No quotes, probably worth linking to the defines for these, you can use the entries in ArduPilot's MAVLink  XML (IE https://mavlink.io/en/messages/ardupilotmega.html#PLANE_MODE )

Reply:
Needs to describe the boolean return value if the mode change failed for any reason.

Reply:
is this correct now?